examples/closure-app/linux/include/ClosureManager.h

```diff
@@ -0,0 +1,175 @@
+/*
+ *
+ *    Copyright (c) 2025 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/*
+ * @class ClosureManager
+ * @brief Manages the initialization and operations related to closure and
+ *        closure panel endpoints in the application.
+ *
+ * @note This class is part of the closure application example
+ */
+
+#pragma once
+
+#include "ClosureControlEndpoint.h"
+#include "ClosureDimensionEndpoint.h"
+
+class ClosureManager
+{
+public:
+    enum Action_t
```

Comment:
enum class is better.

Also constant naming is `kCamelCase`, so these should be `kCalibrateAction` and similar